test/shared/fixtures.ts

```diff
@@ -691,7 +711,15 @@ export const ethVaultFixture = async function (): Promise<EthVaultFixture> {
   // 7. deploy depositDataRegistry
   const depositDataRegistry = await createDepositDataRegistry(vaultsRegistry)
 
-  // 8. deploy implementations and factories
+  // 8. deploy ethValidatorsChecker
+  const ethValidatorsChecker = await createEthValidatorsChecker(
```

Comment:
Please add deployment step to tasks (eth-upgrade, eth-full-deploy-local, eth-full-deploy). For that, please rebase from `eigenlayer` branch